Clean Swift/Unit Tests (Quick-Nimble).xctemplate/___FILEBASENAME___WorkerSpec.swift

```diff
@@ -90,9 +89,37 @@ class ___VARIABLE_sceneName___WorkerSpec: QuickSpec {
                 expect(response.error).toEventually(beNil())
             })
         }
-        
+
+        describe("track analytics") {
+            context("event is screen view", closure: {
```

Comment:
```suggestion
            context("when event is screen view", closure: {
```

- [ ] Should add `when` for BDD consistency

Reply:
OK